addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -110,6 +110,22 @@ $truncate-limits: 2, 3, 10;
     .o_wslides_home_nav {
         top: -40px;
 
+        .navbar {
+            background: white !important;
+
+            .nav-link {
+                color: $navbar-light-color !important;
+
+                &:hover, &:focus {
+                    color: $navbar-light-hover-color !important;
+                }
+
+                &.disabled {
+                    color: $navbar-light-disabled-color !important;
+                }
+            }
+        }
```

Comment:
Should not we just let the navbar have the user menu colors and remove those CSS rules? (and the commit message has to be adapted)

Reply:
![image](https://user-images.githubusercontent.com/71644421/182142833-89538191-b827-45c5-912a-52ba639cb8c2.png)
vs
![image](https://user-images.githubusercontent.com/71644421/182142896-50587a54-f58d-4591-bbba-a40b6b6a0ba0.png)

I don't know... with it having been forced to white before - it sounds like it was a "requirement" (an approved decision).
It was introduced in #31279 (03a085dff0232aa8d76a9280ac19d4615c684b33)
@stefanorigano Is it ok to let that bar follow the theme colors ?

Reply:
Well, of course if you put an ugly green alongside the purple and white design, it seems bad ^^ But in a website where you choose a menu color that goes well with the rest... it is logical to have the courses menu use that color too. Forcing white has just 50% chance of being bad and not going well with the rest of the design. There are only two solutions:

- Let the menu color be the same as the main menu color. Maybe not what the user always wants but 0% chance of having a "very bad" result.

- Allow the admin to control that course menu color (and then we can have it white as a default if we prefer) (which is probably what we should do but best waiting for the (upcoming?) new design system I have to start working on).

Leaving white is just accepting that users cannot use dark color schemes :grimacing: (and it's more CSS rule too so there really is no reason to choose that :grimacing:).

But indeed if @stefanorigano forced white explicitly in a dedicated commit, there may be something I am missing :grimacing: @stefanorigano ?

Reply:
Hello 👋
Sorry for the bad commit message.
Yes, at that time I forced the background to white 'cause, changing palette, the bg was adapting while the links not,  leading to unreadable content with some color combination :/

From what I see in the screenshoots it's not the case anymore (🎊).

> There are only two solutions:
> - Let the menu color be the same as the main menu color. Maybe not what the user always wants but 0% chance of having a "very bad" result.
> - Allow the admin to control that course menu color (and then we can have it white as a default if we prefer) (which is probably what we should do but best waiting for the (upcoming?) new design system I have to start working on).

What about letting the menu color be the same as the main menu color but make it editable?
As an editor this is the behaviour I expect.

Reply:
> What about letting the menu color be the same as the main menu color but make it editable?

That's the solution n°2 I mentioned that we would have to do yes :wink: (I just mentioned we could choose white as a default if *we prefer* but having the menu color as default seems better to me too). I think we better off waiting for the "new system" I mentioned, I'll add this in my todolist. Meanwhile, let's go with solution n°1 @bso-odoo and so remove the CSS rules :+1: